src/actions/auctionActions.js

```diff
@@ -20,24 +20,27 @@ import { formatEth } from '../services/utils';
 import { notify, notifyError } from './notificationActions';
 import _ from 'lodash';
 import { openLogin } from './accountActions';
-import BN from 'bignumber.js'
-
+import BN from 'bignumber.js';
 
 const web3 = new Web3();
 web3.setProvider(new web3.providers.HttpProvider(process.env.PROVIDER_URL || config.providerUrl));
 
 export const fetchNextAuctionDate = () => async dispatch => {
   const engineContract = eth.getEngineContract();
 
-  const {nextStartTimeSeconds} = await engineContract.methods.getNextAuction().call();
+  const { nextStartTimeSeconds } = await engineContract.methods.getNextAuction().call();
 
-  dispatch({ type: FETCH_NEXT_AUCTION_DATE, nextAuctionDate: nextStartTimeSeconds - Date.now() / 1000 })
-}
+  dispatch({
+    type: FETCH_NEXT_AUCTION_DATE,
+    nextAuctionDate: nextStartTimeSeconds - Date.now() / 1000,
+  });
+};
 
 export const fetchBurnedNec = () => async dispatch => {
   const engineContract = eth.getEngineContract();
   const blockRange = await eth.getChartBlockRange(config.chartDuration);
   const burnedNec = [];
+  console.log(blockRange);
```

Comment:
Something that was accidentally left in?

Reply:
Yes, thanks, removed it!